Pop_Quiz.txt

```diff
@@ -13,7 +13,7 @@ Pop Quiz
 6. If not, what should I do?
 
 7. What is the difference between git add . and git commit -a?
-
+Git add . is all files and git commit -a is for those that are tracked.
```

Comment:
Need an empty line between answer and next question

Reply:
Added empty line.